api/design/health_service.go

```diff
@@ -0,0 +1,77 @@
+package design
+
+import (
+	. "goa.design/goa/v3/dsl"
+	cors "goa.design/plugins/v3/cors/dsl"
+	_ "goa.design/plugins/v3/docs"
+)
+
+var _ = Service("health", func() {
+	Description("HealthAPI allows to check if backend data is up-to-date and reliable or not.")
+
+	Error("internal", ErrorResult, "Internal Server Error")
+
+	HTTP(func() {
+		Path("/api/health/v1")
+	})
+	GRPC(func() {
+		Package("api.v1")
```

Comment:
we need full naming for grpc health packages in all ourservices, otherwise if we import things around we have conflict on names
```suggestion
		Package("injective-price-oracle.api.v1")
```